src/store/plugins/pendingTransactionHandler.js

```diff
@@ -38,7 +38,7 @@ export default async (store) => {
     }
   };
 
-  await watchUntilTruthy(() => store.getters['sdkPlugin/sdk']);
+  await watchUntilTruthy(() => store.state.sdkPlugin.ready);
```

Comment:
Why ths change is necessary?

Reply:
In this case `await watchUntilTruthy(() => store.getters['sdkPlugin/sdk'])` never gets resolved

Reply:
Instead I suggest to use the composable `const { getSdk } = useSdk()` and to ensure the sdk is accessible run `await getSdk()`.

Reply:
This is also not working 🥲. It never gets resolved

Reply:
Any errors when doing `getSdk()`? It works perfectly in every other place.

Reply:
No errors, the promise just doesn't resolve

Reply:
OK, for now let's go with your approach. Soon we will remove Vuex.